src/main/java/tc/oc/pgm/restart/RestartManager.java

```diff
@@ -94,6 +96,28 @@ public boolean isRestartDeferredBy(RequestRestartEvent.Deferral deferral) {
     return this.deferrals.contains(deferral);
   }
 
+  public void requestTimedRestart(Duration duration) {
+    Instant queueStamp = Instant.now();
+    long ticks = duration.getStandardSeconds() * 20;
+    timedQueuedRestartTaskID =
+        this.plugin
+            .getServer()
+            .getScheduler()
```

Comment:
`Bukkit.getScheduler()` would make this more concise

Reply:
only did that to stay consistent with https://github.com/Electroid/PGM/blob/2c5a496911b9d3ca8e77ddbb62415c208a70e3b4/src/main/java/tc/oc/pgm/restart/RestartManager.java#L126-L128